client/src/common/components/map/Map.tsx

```diff
@@ -0,0 +1,55 @@
+import "leaflet/dist/leaflet.css";
+
+import L, { Icon } from "leaflet";
+import React from "react";
+import { Map as LeafletMap, Marker, TileLayer } from "react-leaflet";
+
+import markerIconUrl from "../../../assets/images/png/map-marker.png";
+import styles from "./map.module.scss";
+
+// Allow map scroll only inside Helsinki
+const maxBounds: [number, number][] = [[60.33, 25.33], [60.1, 24.73]];
```

Comment:
Would it be possible to use some more readable type for these lat/lng tuples? I have a hard time knowing which value is lat or lng.